package-lock.json

```
@@ -1,24 +1,23 @@
 {
   "name": "pdf-to-img",
-  "version": "4.5.0",
+  "version": "4.4.0",
```

```suggestion
  "version": "5.0.0",
```

this will be breaking change because node18 doesn't support [`getBuiltinModule`](https://nodejs.org/api/process.html#processgetbuiltinmoduleid), so we need to drop support for node v18.

Could you please update the README and CHANGELOG files to reflect this? and `.github/workflows/test.yml`